docs/docs/recoil-sync/api/RecoilURLSyncTransit.md

```diff
@@ -21,51 +21,57 @@ Transit encoding is not as terse or readable as just using [JSON](/docs/recoil-s
 ## Custom Classes
 
 Handlers for custom user classes can be defined with the `handlers` prop:
-```jsx
+
+```tsx
 type TransitHandler<T, S> = {
-  tag: string,
-  class: Class<T>,
-  write: T => S,
-  read: S => T,
+  tag: string;
+  class: Class<T>;
+  write: (T) => S;
+  read: (S) => T;
 };
 ```
 
+Note: ensure the value of the `handlers` prop is a stable constant.
```

Comment:
Probably a good idea to be more forecful with this and include it directly in the main documentation above and not just here by the example.  Maybe something like:
```
  * **`handlers`** - Optional array of handlers for [Custom Classes](#custom-classes).  It is important that this is a stable or memoized array instance.  Otherwise you may miss URL changes as the listener is re-subscribed.